apps/frontend/src/libs/components/balance-wheel-chart/libs/helpers/generate-gradient.helper.ts

```diff
@@ -0,0 +1,53 @@
+import { type ScriptableContext } from "~/libs/types/types.js";
```

Comment:
Add this helper to a separate folder (`generate-gradient-color`). Also the file name should be the same